doc/eclipse.html

```diff
@@ -301,7 +301,8 @@ <h2 align="center">TestNG Eclipse plug-in</h2>
 <li><b>Output directory</b>: the path where to store the output including temp files, report files, etc. By default, the path is relative to each project except you check the option <b>Absolute ouput path</b> below.</li>
 <li><b>Disalbe default listeners</b>: disable the default listeners when launch testng runtime</li>
 <li><b>Show view when test complete</b>: active the TestNG result view when the test complete</li>
-<li><b>Use Project TestNG jar</b>: It's default to be checked which means TestNG Eclipse Plugin will use the project TestNG jar (for example, the TestNG version managed with maven) when launch TestNG runtime. Otherwise, the TestNG eclipse plugin embedded TestNG version will be used.<br/>
+<li><strike><b>Use Project TestNG jar</b></strike>: (<b>NOTE: this property was removed since 6.9.11, which means TestNG plugin won't append the plugin's embedded testng jar to the runtime test process.</b>) <br/>
+It's default to be checked which means TestNG Eclipse Plugin will use the project TestNG jar (for example, the TestNG version managed with maven) when launch TestNG runtime. Otherwise, the TestNG eclipse plugin embedded TestNG version will be used.<br/>
```

Comment:
"It is checked by default, so the TestNG Eclipse plug-in will use the project's `testng.jar` by default"

I don't really understand the rest of the sentence...